### PR TITLE
Cluster nkeys tool

### PR DESCRIPTION
**Pull request by aricart**
cluster jwt generation tool

```
Usage: nrc [-h] [-g] -c <cluster key path> -s <server key/path> [-o <output>]
  -c string
        cluster key path
  -e string
        token expiry (YYYY-MM-DD, 1m, 1h, 1d, 1M, 1y, 1w)
  -g    processArgs cluster and/or server keys
  -o string
        output file (defaults to stdout) (default "--")
  -s string
        server key or path
```

### CODE REVIEW

File: util/nrc/nrc_test.go [hunk 1]
```diff
@@ -0,0 +1,359 @@
+package main
+
+import (
+	"fmt"
+	"github.com/nats-io/jwt"
```
Comment (kozlovic) [inline util/nrc/nrc_test.go]: move those down

File: util/nrc/nrc_test.go [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+		{"3w", time.Now().AddDate(0, 0, 7*3).Unix(), false},
+		{"2M", time.Now().AddDate(0, 2, 0).Unix(), false},
+		{"2y", time.Now().AddDate(2, 0, 0).Unix(), false},
+	}
+	for _, d := range tests {
+		v, err := parseExpiry(d.input)
+		if err != nil && !d.isError {
+			t.Errorf("%s didn't expect error: %v", d.input, err)
+			continue
+		}
```
[the remark this answers is not in the document]
Comment (aricart) [reply util/nrc/nrc_test.go]: obliterated that code somehow

**Comment by kozlovic** [comment]
LGTM

**Comment by derekcollison** [comment]
I assume this can be closed at this point with nsc proper etc, correct?

**Pull request closed without merging**